docs/docs/tutorial.md

```diff
@@ -236,7 +237,7 @@ First, add the third-party library **marked** to your application. This is a Jav
 </head>
 ```
 
-Next, let's convert the comment text to Markdown and output it:
+Let's convert the comment text to Markdown and output it:
```

Comment:
Then we can just put this sentence into the preceding paragraph since they are both super short and flow pretty well.